### PR TITLE
cart and wishlist pages added

### PR DESCRIPTION
**Pull request by Sandesh9834**

### CODE REVIEW

File: Styles/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
+.wishist-cart {
+      display: flex;
+      flex-wrap: wrap;
+      justify-content: space-evenly;
+      padding: 3.1rem;
```
Comment (kushalsourav) [inline Styles/wishlist.css:5]: clean code

**Review by kushalsourav — APPROVED** [review]
looks good to merge

File: Styles/cart.css [hunk 2]
```diff
@@ -0,0 +1,116 @@
+.cart-image {
+      height: 17.5rem;
+      width: 14.3rem;
+    }
+    
+    .cart-heading {
+      text-align: center;
+      margin-top: 20px;
```
Comment (tarunsankhla) [inline Styles/cart.css:8]: Try using rem if above 10px and it would be consistent margin throughout different screen

File: pages/cart.html [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+      </button>
+      <button  class="btn-badge badge-lg">
+        <a href="cart.html">
+          <i class="fas fa-shopping-cart fa-2x"></i>
+        </a>
+        <span class="icon-button-badge flex center icon-xl">10</span>
+      </button>
+  </div>
+</nav>
+<h1 class="cart-heading">My Cart</h1>
```
Comment (tarunsankhla) [inline pages/cart.html:38]: Correct the spacing here and correct intendation

File: pages/cart.html [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    <div class="cart-container">
+      <div>
+        <img
+          class="cart-image"
+          src="sneakers-shoes-adidas-shoes.jpg"
+          alt=""
```
Comment (tarunsankhla) [inline pages/cart.html:44]: Add alt in the images

File: pages/wishlist.html [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+          <button  class="btn-badge badge-lg">
+            <a href="cart.html">
+              <i class="fas fa-shopping-cart fa-2x"></i>
+            </a>
+            <span class="icon-button-badge flex center icon-xl">10</span>
+          </button>
+  </div>
+</nav>
+<h1 class="wishlist-heading center">My Wishlist</h1>
+<div class="wishist-cart flex flex-wrap">
```
Comment (tarunsankhla) [inline pages/wishlist.html:40]: Correct the spacing here too get easier to review html code

**Review by tarunsankhla — APPROVED** [review]
Overall classname and code is great, just minor changes

**Pull request merged**